tests/fsharp/typecheck/sigs/neg04.bsl

```diff
@@ -14,31 +14,31 @@ neg04.fs(26,8,26,17): typecheck error FS0912: This declaration element is not pe
 neg04.fs(32,8,32,11): typecheck error FS0039: The type 'Double' does not define the field, constructor or member 'Nan'. Maybe you want one of the following:
    IsNaN
 
-neg04.fs(46,69,46,94): typecheck error FS0001: Type mismatch. Expecting a
-    ''a * 'b * 'c * 'e'    
-but given a
-    ''a * 'b * 'c'    
-The tuples have differing lengths of 4 and 3
+neg04.fs(46,69,46,94): typecheck error FS0001: Type mismatch. Expecting a tuple of length 4 of type
+    'a * 'g * 'f * 'i    
+but given a tuple of length 3 of type
+    'c * 'l * 'm    
```

Comment:
The additional type checking creates new type variables for the error message. I guess it's correct, because they may be different types than the `'a 'b 'c` that are expected or used elsewhere in code. Hopefully it won't be confusing for anyone.